README.md

```diff
@@ -4,8 +4,17 @@
 
 ## Goal
 
-To be able to run the Digitalmarketplace web service on a single Docker container for 
-exploration, prototyping, development and testing purposes, in a way that is easy also for non-developers.
+To provide a relatively simple way of launching a copy of the Digital Marketplace website loaded with test data.
+
+Some likely users could be:
+
+- people outside the team (and even outside the organisation) who would like to be free to explore how the service works
```

Comment:
Actually, based on your comment, I've removed the first type of users (people outside the team).
I've just tried to launch `dmp-contained` without the data and it doesn't seem to work (I thought it would).

Would you be able to approve the PR now @gidsg ?